src/Kopernicus/packages.config

```diff
@@ -1,11 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Assembly-CSharp" version="1.8.1" targetFramework="net472" />
+  <package id="Defiant_Zombie.KSP.Skeleton.UnityEngine.CoreModule" version="1.8.1" targetFramework="net472" />
```

Comment:
you should not add this

Reply:
This is added because it is a dependency of ParticleSystemModule.